ql/src/java/org/apache/hadoop/hive/ql/exec/vector/expressions/StringGroupConcatColCol.java

```diff
@@ -320,7 +320,8 @@ public void evaluate(VectorizedRowBatch batch) throws HiveException {
       /*
        * Do careful maintenance of the outputColVector.noNulls flag.
        */
-
+      // propagate not nulls flag.
```

Comment:
Above lines says:
     // there are no nulls in either input vector
      /*
       * Do careful maintenance of the outputColVector.noNulls flag.
       */
       
  So the else branch says there are no nulls, it seems strange to then says noNulls = true

Reply:
@zratkai there are two vectors. one is input vector which contains input strings in our case. And the second vector is output which contains the result and flags related to nulls/not nulls. So here the case is if input vector contains mix of NULLs and NOT NULLs then previous steps sets noNulls to false(NULL case) and same flag carried to "else" case[NOT NULL] and we are not setting the **outV.noNulls** flag to true.
I just modified the code to set the **outV.noNulls** flag true in else case and i verified in cluster. Fix is working fine. 
I am trying to add the qtest but i am not able to reproduce the issue locally with q file.

Reply:
@zratkai  yes, but with simple data insert issue is not able to reproduce.